src/main/scala/scalismo/statisticalmodel/experimental/dataset/DataCollectionOfVolumeMesh.scala

```diff
@@ -110,7 +110,7 @@ object DataCollectionOfVolumeMesh {
     val meshFileNames = meshDirectory
       .listFiles()
       .toSeq
-      .filter(fn => fn.getAbsolutePath.endsWith(".vtk") || fn.getAbsolutePath.endsWith(".stl"))
+      .filter(fn => fn.getAbsolutePath.endsWith(".vtk") || fn.getAbsolutePath.endsWith(".vtu")||fn.getAbsolutePath.endsWith(".inp"))
     val (meshes, ioErrors) = DataUtils.partitionSuccAndFailedTries(for (meshFn <- meshFileNames) yield {
       MeshIO.readMesh(meshFn).map(m => TetrahedralMesh3D(m.pointSet, referenceMesh.tetrahedralization))
```

Comment:
I just noticed that calling `DataCollectionOfVolumeMesh.fromMeshDirectory` leads to an empty data collection as the meshes are loaded with the wrong IO function.

```suggestion
      MeshIO.readTetrahedralMesh(meshFn).map(m => TetrahedralMesh3D(m.pointSet, referenceMesh.tetrahedralization))
```